tests/topotests/static_srv6_sids/r1/zebra.conf

```diff
@@ -0,0 +1,19 @@
+hostname r1
+!
+! debug zebra events
+! debug zebra rib detailed
+!
+log stdout notifications
+log commands
+log file zebra.log debugging
+!
+segment-routing
+ srv6
+  locators
+   locator MAIN
+    prefix fcbb:bbbb:1::/48 block-len 32 node-len 16 func-bits 16
+    behavior usid
```

Comment:
Actually, I could get same output without `behavior usid`. Is it normal to do this ?
Sorry again about my limited knowledge.

Reply:
assuming seg6local for compressed and uncompressed results to the same.